VulkanHppGenerator.cpp

```diff
@@ -5049,12 +5049,9 @@ int main(int argc, char **argv)
     template<typename ClassType> ClassType& get() VULKAN_HPP_NOEXCEPT { return static_cast<ClassType&>(*this);}
 
     template<typename ClassTypeA, typename ClassTypeB, typename ...ClassTypes>
```

Comment:
Shouldn't `ClassTypeB` also be removed from template parameter?

Reply:
Thanks for catching this!
But we have to keep ClassTypeB (and use it, of course) to help the compiler to distinguish the different flavors of get().